doc/examples/segmentation/plot_fast_draw_segmentation.py

```diff
@@ -0,0 +1,198 @@
+        rgb_image[yy-brush:yy+brush+1,xx-brush:xx+brush+1] = (255,0,0)
+        # generate labels as foreground, 1
+        labels[yy-brush:yy+brush+1,xx-brush:xx+brush+1] = 1
+    elif event.button == 3:
+        # draw green pixels on the rgb image
+        rgb_image[yy-brush:yy+brush+1,xx-brush:xx+brush+1] = (0,255,0)
+        # generate labels as background, 2
+        labels[yy-brush:yy+brush+1,xx-brush:xx+brush+1] = 2
+    # refresh plot
+    pltimg.set_data(rgb_image)
+    plt.draw()
+    
+# keyboard callback events
+def onKeypress(event):
+    """This function is called when keyboard buttons are pressed
+    """
+    global target_label
+    if event.key == ' ':
+        # if space bar is pressed, FastDRaW  and RW segmentations are performed
+        
+        # FastDRaW
+        t = time.clock()
+        # --- perform the FastDRaW segmentation
+        segm = fastdraw.update(labels, target_label=target_label, k=0.5)
+        # --- display the contour in red over the original image
+        contour = (dilation(segm, disk(1)) - segm).astype(np.bool)
+        result = original_image.copy()
+        result[contour,:] = (255,0,0)
+        # --- show the image
+        ax2.imshow(result)
+        ax2.set_title("FastDRaW time "+str(time.clock()-t)+" s")
+        plt.draw()
+        
+        # RW
+        t = time.clock()
+        # perform the RW segmentation
+        segm = random_walker(image,labels,beta=beta)
+        # select which label to display
+        # since this is a binary case segmentation (foreground/background),
+        # segm == 1 is similar to segm == 2
+        segm = segm == 1 
+        # --- display the contour in red over the original image
+        contour = (dilation(segm, disk(1)) - segm).astype(np.bool)
+        result = original_image.copy()
+        result[contour,:] = (255,0,0)
+        # --- show the image
+        ax3.imshow(result)
+        ax3.set_title("RW time "+str(time.clock()-t)+" s")
+        plt.draw()
+    
+
+"""
+Main
+----
+"""
+# load array like image
+image = data.coins()
+# convert image to rgb for drawing purposes
+if image.ndim == 2:
+    rgb_image = np.dstack((image,image,image))
+# create a copy of the rgb image to display the results
+original_image = rgb_image.copy()
+# initialize labels to zeros
+labels = np.zeros_like(image)
+
+# display the original image, on which the user can draw
+fig = plt.figure()
+ax1 = fig.add_subplot(131)
+pltimg = plt.imshow(rgb_image)
+ax1.set_title('Original image')
+plt.axis('off')
+
+# set callback functions
+did = fig.canvas.mpl_connect('motion_notify_event', onDraw)
+cid = fig.canvas.mpl_connect('button_press_event', onClick)
+kid = fig.canvas.mpl_connect('key_press_event', onKeypress)
+
+# display FastDRaW result image
+ax2 = fig.add_subplot(132)
+ax2.imshow(original_image)
+ax2.set_title('FastDRaW')
+plt.axis('off')
+
+# display Random Walker result image
+ax3 = fig.add_subplot(133)
+ax3.imshow(original_image)
+ax3.set_title('RW')
+plt.axis('off')
```

Comment:
Have a look at our other examples. We've mostly standardised around `fig, axes = plt.subplots(1, 3)`. And I think `axes[0, 2].axis('off')` works, but I might be wrong there.